packages/@okta/vuepress-site/docs/reference/api/users/index.md

```diff
@@ -1163,7 +1163,7 @@ The first three parameters in the table below correspond to different ways to li
 
 - If you don't specify a value for `limit`, the maximum (200) is used as a default.  If you are using a `q` parameter, the default limit is 10.
 - An HTTP 500 status code usually indicates that you have exceeded the request timeout. Retry your request with a smaller limit and [paginate](/docs/reference/core-okta-api/#pagination) the results.
-- Okta strongly advises that you use the `search` parameter, which delivers optimal performance. The `q` or `filter` parameters may struggle to perform, in which case Okta recommends reformatting the request to use `search`.
+- The `search` parameter delivers optimal performance. Using the `q` or `filter` parameters may yield no results, or may cause the search to struggle to perform, in which case you should try reformatting the request to use `search`.
```

Comment:
```suggestion
- The `search` parameter delivers optimal performance. Using the `q` or `filter` parameters may yield no results. It may also affect search performance, in which case you should try reformatting the request to use `search`.
```